ui/src/components/song/SongListContainer.tsx

```diff
@@ -72,61 +107,49 @@ const SongListContainer: FC = (): ReactElement => {
   };
 
-  useEffect(() => {
-    getSongResults();
-  }, [getSongResults]);
+  const [viewOption, setViewOption] = useState<string>('cards');
 
   return (
     <>
-      <Container fixed sx={{ padding: '5em 24px 24px', height: '100%' }}>
-        {/* Toolbar at the top */}
-        <Stack
-          direction="row"
-          display="flex"
-          justifyContent="space-between"
-          pb="10px"
-          pl={{ base: '0', md: '15px' }}
-        >
-          {/* Title */}
-          <Typography variant="h1" color="white" sx={{ display: 'flex', alignItems: 'center' }}>
-            <MusicNote
+      <Container
+        fixed
+        sx={{ py: '1rem', px: '1.5rem', height: '100%', minWidth: '100%', overflow: 'auto' }}
+      >
+        <PageHeader
+          title="Songs"
+          icon={<MusicNote />}
+          actionButtons={
+            <Button
+              variant="outlined"
               sx={{
-                color: 'primary.light',
-                backgroundColor: 'primary.dark',
-                borderRadius: '50%',
-                width: '2em',
-                height: '2em',
-                padding: '0.5em',
-                mr: 3,
+                border: 0,
+                padding: '10px 25px',
+                borderRadius: '40px',
+                backgroundColor: '#D0BCFF',
+                color: '#381E72',
+                textTransform: 'none',
+                '&:hover': {
+                  backgroundColor: '#D0BCFF',
+                  opacity: '0.95',
+                },
+                transition: 'all 0.1s ease-in-out',
               }}
-            />
-            Songs
-          </Typography>
-
-          {/* Add new song button */}
-          <Button
-            variant="outlined"
-            sx={{
-              borderWidth: '2px',
-              padding: '10px 25px',
-              borderRadius: '40px',
-              backgroundColor: 'rgba(208, 188, 255, 0.12)',
-              color: '#D0BCFF',
-              textTransform: 'none',
-            }}
-            startIcon={<AddIcon />}
-            onClick={() => navigate('/song/add')}
-          >
-            <Typography variant="subtitle1">New Song</Typography>
-          </Button>
-        </Stack>
+              startIcon={<Add />}
+              onClick={() => navigate('/song/add')}
+            >
+              <Typography variant="subtitle1" fontWeight={700}>
+                New Song
+              </Typography>
+            </Button>
+          }
+        />
 
         {/* TODO: Mobile Search bar and filters */}
         <Box display={{ base: 'block', md: 'none' }}></Box>
 
-        {/* Desktop search bar and filters */}
-        <Stack direction="row" width="100%" gap={3}>
-          <Box display={isDesktop ? 'block' : 'none'}>
+        {/* Desktop filter menu */}
+        <Stack direction="row" maxWidth="100%" width="100%" gap={'1%'}>
+          <Box display={isDesktop ? 'flex' : 'none'} minWidth="28%" width="28%">
```

Comment:
It's never really a good idea in the long run if we use very specific number for width like this. Prone to breaking for diff screen sizes. If u have time can try play with the flex or use Grid.